en/docs/guides/identity-verification/idvp-ui.md

```diff
@@ -0,0 +1,307 @@
+- Configuration section in the IDV Provider creation wizard in the console.
+- Settings section in the IDV Provider edit page in the console.
+- Configuration Properties section in the IDV Provider create/edit pages in the management console.
+
+The UI metadata defines what content needs to be rendered, how it should be structured, and what validations need to be performed.
+
+**Structure**
+
+The metadata.json file should contain the following structure.
+```
+{
+    "common": {
+    "configProperties": [ <An array of input fields> ]
+    }
+}
+```
+**Supported Input Types**
+
+Currently the following input field types are supported.
+
+| Input Field Type | Description                                                                          | Supported In |
+|------------------|--------------------------------------------------------------------------------------|--------------|
+| identifier       | Renders a text input field that performs validations for identifiers by default.     | Console      |
+| number           | Renders a numeric input field.                                                       | Console      |
+| resource_name    | Renders a text input field that performs validations for resource names by default.  | Console      | 
+| client_id        | Renders a text input field that performs validations for client ids by default.      | Console      |
+| description      | Renders a text area field that performs validations for descriptions by default.     | Console      |
+| email            | Renders a text input field that performs validations for email addresses by default. | Console      |
+| url              | Renders a text input field that performs validations for URLs by default.            | Console      |     
+| password         | Renders a password input field with a show/hide button.                              | Console      |
+| text_area        | Renders a text area field.                                                           | Console      |
+| checkbox         | Renders a checkbox.                                                                  | Console      |
+| toggle           | Renders a toggle button.                                                             | Console      |
+| dropdown         | Renders a dropdown.                                                                  | Console      |
+| default          | Renders a text input field.                                                          | Console      |
+
+Refer [validation.ts](https://github.com/wso2/identity-apps/blob/master/modules/validation/src/validation.ts) and 
+[dynamic-ui-helper.tsx](https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/identity-verification-providers/components/forms/helpers/dynamic-ui-helper.tsx#L269) for more details on how the validations are performed.
+
+**Supported Attributes**
+
+Common Attributes - All Input Fields
```

Comment:
`Common Attributes - All Input Fields` should be a heading.

Reply:
Resolved